editor/editor_node.cpp

```diff
@@ -2540,6 +2542,32 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 	}
 }
 
+void EditorNode::_request_screenshot() {
+	_screenshot();
+}
+
+void EditorNode::_screenshot(bool p_use_utc) {
+	Dictionary datetime = _OS::get_singleton()->get_datetime(p_use_utc);
```

Comment:
`_OS` is not meant to be used from C++, it's only used for bindings. You should use `OS` methods.

Reply:
Resolved, it does feel like a lot of duplicated code though..

Reply:
Well, I didn't mean that you should reimplement the feature here... But it should be moved from `_OS` to `OS` if it's useful also to engine code. `_OS` should only have bindings.

Reply:
Well this should be using an ISO 8601 string format, but that doesn't play well with the operating system file limits. So we're using this custom format.

Reply:
I'll code up something like ISO 8601 format (2013-04-01T13:01:02) that then replaces ":" with "".

Reply:
Up for review.